runtime-modules/content-working-group/src/lib.rs

```diff
@@ -2033,6 +2041,41 @@ decl_module! {
             let mint = <minting::Module<T>>::mints(mint_id); // must exist
             let new_capacity = mint.capacity() + additional_capacity;
             let _ = <minting::Module<T>>::set_mint_capacity(mint_id, new_capacity);
```

Comment:
I recommend here to add "?":
```
<minting::Module<T>>::set_mint_capacity(mint_id, new_capacity)?;
```
Since set_mint_capacity() returns the `Result` and it's possible that another error would be added later inside it - it's safer to process this error rather than ignore it.

Reply:
That is a good idea.

Reply:
done in 1d115a84a74548faa113f678570de987e90417b4